index.php

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Armadillo Estates</title>
+    <link rel="stylesheet" href="./css/bootstrap.min.css" type="text/css">
+</head>
+<body>
+<nav class="navbar navbar-light bg-light mb-4">
+    <div class="container">
+        <a class="navbar-brand" href="#">
+            <img src="./img/logo.png" height="40"
+                 class="d-inline-block align-top" alt="">
+        </a>
+        <h2>Armadillo Estates</h2>
+        <h2></h2>
```

Comment:
What's this for?

Reply:
The parent element has justify-content spacebetween on it. And its the only easy way for having the Logo on the left and then title in the middle at the top of the page